tests/check_add_email.py

```diff
@@ -16,14 +16,14 @@
 class TestAddEmail(BaseTest):
 
     def test_add_email(self, base_url, selenium, timeout, verified_user, new_email):
-        browser_id = BrowserID(selenium, timeout)
-        browser_id.sign_in(verified_user['email'], verified_user['password'])
+        bidpom = BIDPOM(selenium, timeout)
+        bidpom.sign_in(verified_user['email'], verified_user['password'])
```

Comment:
Despite my original reaction, I actually think this is fine. In tests we could choose to name this `browser_id` or even `persona` for improved readability, but it's not something we need to do here.